aspnet-core/RadialMenu/TemplateSupport.md

```diff
@@ -1,17 +1,17 @@
 ---
 layout: post
-title: Syncfusion Radial Menu Template Support.
-description: template support.
+title: Template Support | ASP.NET Core Radial Menu | Syncfusion
```

Comment:
Template Support in ASP.NET Core Radial Menu Control | Syncfusion

Reply:
Added